Yoast/wordpress-seo
# Add go premium box

## Summary
Introduces the Premium upsell box below the settings on the settings pages of the plugin.

This PR can be summarized in the following changelog entry:
- Adds a WordPress SEO Premium motivation box on Yoast SEO settings pages.

## Relevant technical choices:

* Allowed to pass the hook on which the block will be displayed
* Added the SCSS to the yst_plugin_tools file, as this is loaded on all needed pages and contains sidebar styling

## Test instructions

This PR can be tested by following these steps:

* Go to an admin page of Yoast SEO and see the admin box below the settings

Fixes #6785
